firebase.json

```diff
@@ -0,0 +1,57 @@
+{
+  "hosting": {
```

Comment:
Ajouter une section rewrites, vu que c'est une SPA ?

```json
{
    "rewrites": [
      {
        "source": "**",
        "destination": "/index.html"
      }
    ],
}
```

Reply:
Not needed 'cause it's not a SPA, Gatsby is a static site generator ;)